crt_portal/static/sass/custom/landing.scss

```diff
@@ -154,10 +154,11 @@ $arrow-height: 24px;
           height: 1rem;
         }
 
+        // Override USWDS styles
         a {
-          color: #162e51;
-          padding-top: 0px;
-          padding-left: 0.5rem;
+          color: #162e51 !important;
+          padding-top: 0 !important;
+          padding-left: 0.5rem !important;
```

Comment:
Could this be done by narrowing the selector?

Reply:
Not really - at least, not in a way that's even worse-looking than `!important` - although it's worth looking at why the thing I'm overriding needs to exist. I might need to redo part of the navigation and unwind some of the complexity that's already there. I'll take a closer look.